spec/index.html

```diff
@@ -7410,6 +7410,127 @@ <h5>TZ</h5>
+                  BIND( TRIPLE(?s, ?p, ?o) AS ?qt )
+                  ?qt :tripleAdded ?date
+              }
+            </pre>
+          </section>
+
+          <section id="func-subject">
+            <h5>SUBJECT</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">SUBJECT</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+	            If the argument is a
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>, 
+              the function returns the 
+              <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>
+              of the quoted triple. 
+              If the argument is not a 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              an error is raised.
+            </p>
+          </section>
+          
+          <section id="func-predicate">
+            <h5>PREDICATE</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">PREDICATE</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+              If the argument is a
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              the function returns the 
+              <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>
+              of the quoted triple.
+              If the argument is not a 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              an error is raised.
+            </p>
+          </section>
+
+          <section id="func-object">
+            <h5>OBJECT</h5>
+            <pre class="prototype nohighlight"><span class="return">RDF term</span>  <span class="operator">OBJECT</span> (<span class="type RDFterm">quoted triple</span> <span class="name">quoted-triple</span>)</pre>
+            <p>
+              If the argument is a
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              the function returns the 
+              <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>
+              of the quoted triple.
+              If the argument is not a 
+              <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>,
+              an error is raised.
+            </p>
+          </section>
+
+          <section>
+            <h5>isTRIPLE</h5>
+            <pre class="prototype nohighlight"><span class="return">xsd:boolean</span>  <span class="operator">isTRIPLE</span> (<span class="type RDFterm">RDF term</span> <span class="name">term</span>)</pre>
+            <p>
+              If the argument is a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>, 
+              the function returns true.
+              Otherwise, if the argument is any other kind of 
```

Comment:
```suggestion
              If the argument is any other kind of 
```

Reply:
"Otherwise" is intended - it is broad "everything else". (IMO We ought to be using this pattern in more places).

Reply:
OK ... see [new suggestion](https://github.com/w3c/sparql-query/pull/92/files#r1242924657).